tools/ustat.py

```diff
@@ -239,10 +240,10 @@ def _loop_iter(self):
             counts.update(probe.get_counts(self.bpf))
             targets.update(probe.targets)
         if self.args.sort:
-            counts = sorted(counts.items(), key=lambda _, v:
-                            -v.get(self.args.sort.upper(), 0))
+            counts = sorted(counts.items(), key=lambda kv:
+                            -kv[1].get(self.args.sort.upper(), 0))
         else:
-            counts = sorted(counts.items(), key=lambda k, _: k)
+            counts = sorted(counts.items(), key=itemgetter(1))
```

Comment:
This is cute but maybe for simplicity stick to kv[1]? People reading and maintaining these tools aren't likely to be native pythonistas (myself included)...